content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Dapatkan _Shell_ Untuk Masuk ke Kontainer yang Sedang Berjalan
```

Comment:
You can't use HTML or Markdown in titles
```suggestion
title: Dapatkan Shell Untuk Masuk ke Kontainer yang Sedang Berjalan
```

Reply:
Sorry... my bad! Thanks @sftim.